src/brevitas/graph/quantize.py

```diff
@@ -263,7 +264,11 @@ def preprocess_for_quantize(
         equalize_merge_bias=True,
         merge_bn=True,
         equalize_bias_shrinkage: str = 'vaiq',
-        equalize_scale_computation: str = 'maxabs'):
+        equalize_scale_computation: str = 'maxabs',
+        channel_splitting=False,
```

Comment:
Let's use the ratio to define whether this is enabled or not. Channel split = 0 is disabled.
Default ratio I would put disabled since channel splitting does not come for free (unlike weight eq for example)